modules/mesh-layers/package.json

```diff
@@ -33,6 +33,8 @@
   },
   "dependencies": {
     "@loaders.gl/gltf": "^3.0.8",
+    "@luma.gl/constants": "^8.5.10",
+    "@luma.gl/core": "^8.5.10",
```

Comment:
What prompted the dependency change? luma.gl core should be peer dependency even if it's needed. Conflicting copies will break the build.